### PR TITLE
Remove rimraf as dependency

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: test/unit-tests/yok.ts [hunk 1]
```diff
@@ -321,6 +321,12 @@ describe("yok", () => {
 		});
 	});
 
+    function deleteDirectory(directory: string): boolean {
+        shelljs.rm("-rf", directory);
+        let err = shelljs.error();
```
Comment (rosen-vladimirov) [inline test/unit-tests/yok.ts]: mixed spaces and tabs

File: test/unit-tests/yok.ts [hunk 2]
```diff
@@ -339,11 +345,9 @@ describe("yok", () => {
 		// Get the real instance here, so we can delete the file before asserts.
 		// This way we'll keep the directory clean, even if assert fails.
 		let resultFooObject = injector.publicApi.foo;
-		rimraf(filepath, (err: Error) => {
-			if(err) {
-				console.log(`Unable to delete file used for tests: ${filepath}.`);
-			}
-		});
+		if (!deleteDirectory(filepath)) {
+            console.log(`Unable to delete file used for tests: ${filepath}.`);
```
Comment (rosen-vladimirov) [inline test/unit-tests/yok.ts]: mixed tabs and spaces

File: file-system.ts [hunk 3]
```diff
@@ -132,14 +132,15 @@ export class FileSystem implements IFileSystem {
...
 		let future = new Future<void>();
-		rimraf(directory, (err:Error) => {
-			if(err) {
-				future.throw(err);
-			} else {
-				future.return();
-			}
-		});
 
+        shelljs.rm("-rf", directory);
```
Comment (rosen-vladimirov) [inline file-system.ts]: mixed tabs and spaces

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**